src/components/KeyValueInput/KeyValueInput.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div data-testid="uic-key-value-input">
+    <fieldset class="w-full">
+      <legend class="uic-label">{{ label }}</legend>
+      <ul class="flex flex-col gap-4 w-full">
+        <li
+          v-for="([key, value], index) in modelValue"
+          :key="`key-value-${index}`"
+          class="grid gap-6 items-center"
+          style="grid-template-columns: 1fr 1fr min-content"
+        >
+          <TextInput
+            :id="`key-${index}`"
+            :data-testid="`uic-key-value-input-key-${index}`"
+            :error="Boolean(keyErrors?.[index])"
+            :helper-text="keyErrors?.[index]"
```

Comment:
<img width="530" alt="image" src="https://github.com/lob/ui-components/assets/83967528/880fea11-ef47-4b41-b344-49e603ea23d4">

what are some possible `keyErrors`? I couldn't generate one

Reply:
No errors are generated with the component, that is to be handled by the implementation. I thought about adding errors but ultimately decided that the validity is very dependent on the implementation and thus the implementation should handle all validation